# Generated from https://pypi.org/project/datadog/
package:
  name: py3-datadog
  version: 0.47.0
  epoch: 0
  description: The Datadog Python library
  copyright:
    - license: BSD-3-Clause
  dependencies:
    runtime:
      - py3-requests
      - py3-typing
      - python3

environment:
  contents:
    packages:
      - build-base
      - busybox
      - ca-certificates-bundle
      - py3-gpep517
      - py3-pip
      - py3-setuptools
      - py3-wheel
      - python3
      - wolfi-base

pipeline:
  - uses: fetch
    with:
      expected-sha256: 47be3b2c3d709a7f5b709eb126ed4fe6cc7977d618fe5c158dd89c2a9f7d9916
      uri: https://files.pythonhosted.org/packages/source/d/datadog/datadog-${{package.version}}.tar.gz

  - runs: |
      # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
      export SOURCE_DATE_EPOCH=315532800
      python3 -m pip install -U hatchling hatch-vcs
      python3 -m gpep517 build-wheel \
        --wheel-dir dist \
        --output-fd 3 3>&1 >&2
      python3 -m installer \
        -d "${{targets.destdir}}" \
        dist/*.whl

  - uses: strip

update:
  enabled: true
  release-monitor:
    identifier: 35391
